src/ros_filter.cpp

```diff
@@ -3330,15 +3260,9 @@ bool RosFilter<T>::revertTo(const rclcpp::Time & time)
 
     if (last_history_state) {
       RF_DEBUG(
-        "Will revert to oldest state at " <<
-          filter_utilities::toSec(last_history_state->latest_control_time_) <<
-          ".\n");
-
-      // ROS_WARN_STREAM_DELAYED_THROTTLE(history_length_, "Could not revert "
```

Comment:
I think I put a bunch of these back and just used the `RCLCPP` equivalents in the `rolling` branch.